templates/_schemas/definitions/person.twig

```diff
@@ -0,0 +1,10 @@
+{% macro person(props, schema = null) %}
+  
+  {% set schema = schema ?? craft.schema.Person %}
+  {% set schema = schema
+    .name(props.name)
```

Comment:
Should inherit `Thing` and these properties will be auto-set by `Thing`.

Reply:
Done.